packages/integrations/vercel/test/edge-middleware.test.js

```diff
@@ -4,27 +4,38 @@ import chaiJestSnapshot from 'chai-jest-snapshot';
 
 use(chaiJestSnapshot);
 
-describe('Serverless prerender', () => {
-	/** @type {import('./test-utils').Fixture} */
-	let fixture;
+before(function () {
+	chaiJestSnapshot.resetSnapshotRegistry();
+});
 
-	beforeEach(function () {
-		chaiJestSnapshot.configureUsingMochaContext(this);
-	});
+beforeEach(function () {
+	chaiJestSnapshot.configureUsingMochaContext(this);
+});
```

Comment:
Do we still need this code since there's the `setup.js` file now?

Reply:
Oh yeah, good catch. We can remove it